lib/chewy/index/adapter/orm.rb

```diff
@@ -98,7 +98,7 @@ def import_fields(*args, &block)
 
         def load(ids, **options)
           scope = all_scope_where_ids_in(ids)
-          additional_scope = options[options[:_index].to_sym].try(:[], :scope) || options[:scope]
+          additional_scope = options[options[:_index].try(:to_sym)].try(:[], :scope) || options[:scope]
```

Comment:
Just curious: where/why did it fail?

Reply:
I had failures in `load_descendants`, but I fixed the way we use `load` there.